bigquery_etl/view/generate_stable_views.py

```diff
@@ -117,16 +167,12 @@ def write_view_if_not_exists(target_project: str, sql_dir: Path, schema: SchemaF
         / schema.bq_dataset_family
         / schema.bq_table_unversioned
     )
+
     target_file = target_dir / "view.sql"
 
     if target_file.exists():
         return
 
-    # Exclude doctypes maintained in separate projects.
-    for prefix in SKIP_PREFIXES:
-        if schema.bq_dataset_family.startswith(prefix):
-            return
-
```

Comment:
Moved this condition into the `get_stable_table_schemas` function so that it has broader effect.